MobileSDK_To_AmplifySwift_Guidance.md

```diff
@@ -0,0 +1,162 @@
+| ----------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Invoke](https://docs.amplify.aws/gen1/swift/sdk/api/rest/) | [Create](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/set-up-rest-api/#make-a-post-request) / [Fetch](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/fetch-data/) / [Update](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/update-data/) / [Delete](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/delete-data/) |
+| [IAM Auth](https://docs.amplify.aws/gen1/swift/sdk/api/rest/#iam-authorization)                                                    | [Configure IAM](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/customize-authz/#iam-authorization)                                                                                                                                                                                                                                                         |
+| [Cognito User Pools](https://docs.amplify.aws/gen1/swift/sdk/api/rest/#cognito-user-pool-authorization)                                          | [CUP Auth](https://docs.amplify.aws/gen1/swift/build-a-backend/restapi/customize-authz/#cognito-user-pool-authorization)                                                                                                                                                                                                                                                |
+
+---
+
+### GraphQL API
+
+(Replace **AWSAppSyncClient**)
+
+| AWS Mobile SDK for iOS | Amplify Swift                                                                                                           |
+| ---------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| Queries / Mutations    | [Create / Read / Update / Delete](https://docs.amplify.aws/gen1/swift/build-a-backend/graphqlapi/create-update-delete/) |
+| Subscriptions          | [Real‑time Subscribe](https://docs.amplify.aws/gen1/swift/build-a-backend/graphqlapi/subscribe-to-data/)                |
+| Auth Modes             | [Configure Auth Modes](https://docs.amplify.aws/gen1/swift/build-a-backend/graphqlapi/configure-auth-modes/)            |
```

Comment:
first link: https://docs.amplify.aws/gen1/swift/sdk/api/graphql/#authorization-modes
second link is broken : https://docs.amplify.aws/gen1/swift/build-a-backend/graphqlapi/customize-authz-modes/